### PR TITLE
Fix SPI compatibility test to not use latest release

### PR DESCRIPTION
**Pull request by electrum**
This makes the test deterministic and allows running on older branches.

## Documentation

(x) No documentation is needed.

## Release notes

(x) No release notes entries required.

### CODE REVIEW

File: core/trino-server-rpm/pom.xml [hunk 1]
```diff
@@ -116,7 +116,6 @@
             <plugin>
                 <groupId>org.codehaus.gmaven</groupId>
                 <artifactId>groovy-maven-plugin</artifactId>
-                <version>2.1.1</version>
```
Comment (kokosing) [inline core/trino-server-rpm/pom.xml:119]: separate commit?

File: core/trino-spi/pom.xml [hunk 2]
```diff
@@ -117,6 +117,7 @@
                     <include>io/trino/spi/trino-spi-version.txt</include>
                 </includes>
             </resource>
+
```
Comment (kokosing) [inline core/trino-spi/pom.xml:120]: separate commit?

**Review by kokosing — APPROVED** [review]
Thanks

File: core/trino-server-rpm/pom.xml (hunk 1 above)
Comment (electrum) [reply core/trino-server-rpm/pom.xml:119]: This just moves the version to the root POM.

**Pull request merged**